### DISCUSSION TITLE
Windows on Arm support?

### DISCUSSION THREAD

**Discussion by audioeng89** [jeffvli/feishin#726] [Ideas]
Possible to compile Feishin for Arm-based Windows PCs natively?
(Performance under emulation is a bit so-so).

**Comment by jeffvli** [comment]
If you're willing to fork and try building the arm64 version yourself to ensure that it works properly, I can probably add it to the release builds. Arm on windows is niche enough that it's not something I want to test and support without having someone else test first.

Use the `package` build in the package.json scripts and add the `--arm64` flag to the end of it. 
Reference: https://www.electron.build/cli.html

**Reply by audioeng89** [reply]
OK. Feel a bit out of my depth, but I've got 8.1 running natively on WoA.
Oddly, I didn't need to touch the package.json. It seems like it compiled for Arm automatically because NPM detected I'm on an Arm machine? Anyway, it's definitely running without emulation now.

Initial impressions are good. CPU usage is down, UI is much more responsive like back on my x64 laptop, no playback stutters so far. Will give it a good test today and let you know how it goes.
